package.json

```diff
@@ -20,7 +20,7 @@
     "vibe-check": "npx vite-bundle-visualizer",
     "generate-openapi-client": "openapi-generator-cli generate -i osidb-3.4.0.oas3.json --output src/generated-client -g typescript-axios",
     "mock-osidb-server": "prism mock osidb-3.0.0.oas3.json",
-    "lint": "eslint . --ext .js,.ts,.vue"
+    "lint": "eslint . --ext .js,.ts,.vue --max-warnings=0"
```

Comment:
In my mind, the warnings give valuable feedback and my preference would be to keep them

Reply:
Everything is going to be preserved, this is just a way how to treat every warning as error in the eslint, it basically just tells the linter that any warning is unacceptable and linter returns the error code.

Reply:
Ok, I understand, so probably lets remove this and declare some things as errors as I've suggested? I was thinking at least about indentation as the inconsistencies on that are sometimes really crazy, wdyt?

Reply:
That sounds great